examples/usbboot.ts

```diff
@@ -13,19 +13,61 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { argv } from 'process';
+
+/**
+ * Unlocking with secureboot
+ *
+ * If you need to unlock an cm4 with secureboot, you need to provide a signed boot-image.
+ * Pass the bootImageFolder flag with the path to the folder containing the signed boot-image
+ */
+
 import ProgressBar = require('progress');
 
 import { scanner, sourceDestination } from '../lib/';
 
 import { pipeSourceToDestinationsWithProgressBar } from './utils';
 
+// Parse command line arguments
+const args = process.argv.slice(2); // removes 'node' and the script name from the args
+const flags: any = {};
+
+args.forEach((arg: string, index: number) => {
+	// Check if the argument is a flag in the format --flag=value
+	if (arg.startsWith('--')) {
+		const key: string = arg.substring(2);
+		const value: string = args[index + 1];
+		flags[key] = value;
+	}
+});
+
+if (!flags.source) {
+	console.log("No source has been provided, won't try to flash anything");
+}
+
+if (flags.bootImageFolder !== '') {
+	console.log(`Using external folder ${flags['bootImageFolder']}`);
+}
+
+if (flags.help) {
+	console.log(
+		'Usage: ts-node usbboot.js --bootImageFolder <bootImageFolder> --source <image>',
+	);
+	console.log(
+		'Beware, `source` image will be flashed to all USBboot devices, so make sure you know what you are doing',
+	);
+	console.log(
+		'To unlock a secureboot CM4, set the bootImageFodler to the folder containing a signed boot-image and config.txt',
```

Comment:
typo: bootImageFodler

Could we not make it device specific so we don't have the change it later? Also, `unlock` gives the impression a device can be unlocked which is not the case.

Maybe something like:

> To expose the internal mass storage device on a locked device, set the `bootImageFolder` to the path containing a signed `boot.img` and `config.txt`.

nitpick: I prefer the Unix-speak `directory` to the Windows-speak `folder`

Reply:
Good point; I changed the wording.